students/ZidanLuo/session01/break_me.py

```diff
@@ -0,0 +1,18 @@
+def NameError ():
+    print(x)
+
+# Passing arguments of the wrong type should result in a TypeError
+def TypeError(x):
+    x = x + 1
+
+def SyntaxError():
+    print("error" + 1 + ' ')
+
+def AttributeError(x,y):
```

Comment:
this doesn't raise an Attribute Error.

If you call it with one argument, you get a TypeError